cla_public/apps/base/tests/test_locale.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+from cla_public.apps.base.views import set_locale
+
+from cla_public.app import create_app
+from flask import request
+
+class LocaleTest(unittest.TestCase):
+
+  def setUp(self):
+    self.app = create_app('config/testing.py')
+    ctx = self.app.test_request_context()
+    ctx.push()
+    self.client = self.app.test_client()
+
+  def test_locale_cookie_is_set(self):
+    with self.app.test_client() as client:
+      response = client.get('/?locale=en_GB')
+      self.check_cookie(response, 'locale', 'en_GB')
```

Comment:
It feels we should also test the redirection here. What do you think?

Reply:
Sure.

Reply:
Will add this to a tech debt ticket as discussed.